docs/src/main/asciidoc/_elytron/Elytron_Subsystem.adoc

```diff
@@ -772,97 +754,12 @@ and openssl provider loaders.
 
 [[default-application-authentication-configuration]]
 == Default Application Authentication Configuration
-
 By default, applications are secured using legacy security domains.
 Applications must specify a security domain in their _web.xml_ as well
 as the authentication method. If no security domain is specified by the
 application, WildFly will use the provided _other_ legacy security
 domain.
 
-[[update-wildfly-to-use-the-default-elytron-components-for-application-authentication]]
-=== Update WildFly to Use the Default Elytron Components for Application
```

Comment:
I think it is worth to have similar section using ApplicationDomain security domain. Which would be current alternative to removing section.

Reply:
In the using Elytron section I also added some documentation regarding the application-security-domain resource in the Undertow subsystem - it may be possible to cross reference that as well.

Reply:
Modified the section to reference ApplicationDomain/Darren's documentation regarding application-security-domain